src/Sylius/Bundle/AdminApiBundle/composer.json

```diff
@@ -24,15 +24,15 @@
     ],
     "require": {
         "php": "^7.3",
-        "friendsofsymfony/oauth-server-bundle": "^1.6",
+        "friendsofsymfony/oauth-server-bundle": "^1.6 || ^2.0@dev",
```

Comment:
As far as I see, it breaks the build. The problem is, we have a spec for the `ClientManager` class that extends the `FOS\OAuthServerBundle\Entity\ClientManager`. In `^1.6` it requires `EntityManager` from `Doctrine\Common\Persistance`, but in `^2.0@dev` it wants it from new `Doctrine\Persistence` component. For some reason, the `1.6.2` version is installed in the build, not `2.0` - which results in an error.

It raises the question, can we even say this bundle is working with the `^1.6` version of this dependency? It was not a problem in the rest of the bundles, so should not be here either 🤷‍♂️

Reply:
It is working with the current version, as well as with 2.0 - just the spec is something that we can't really fix, so I removed it, as it wasn't testing anything else than proxying a call in the end.